### PR TITLE
[minor] Add OCP 4.14 support by default

### PR DESCRIPTION
**Pull request by istrate**
## Description

Work part of adding support OCP 4.13.x and OCP 4.14.x
Making OCP 4.14 version as default

**Note to reviewers: Not to be merged until after final decision on when to GA OCP 4.13 and 4.14 support in MAS**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ It is possible to develop the Ansible roles without needing to build the collect
   vars:
     cluster_type: rosa
     cluster_name: fvtrosa
-    ocp_version: "4.10.17"
+    ocp_version: "4.14.5"
```
Comment (sanju7216) [inline CONTRIBUTING.md:80]: i think we have moved to 4.14.12

File: docs/index.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ ansible-galaxy collection install ibm.mas_devops
 
 Optionally, you can also pin the version of the collection that you install, allowing you to control exactly what version of the collection is in use in your automation:
 ```
-ansible-galaxy collection install ibm.mas_devops:10.6.2
+ansible-galaxy collection install ibm.mas_devops:18.10.4
```
Comment (sanju7216) [inline docs/index.md:88]: 18.17.0 will support ocp 4.13 and 4.14

File: ibm/mas_devops/playbooks/ocp_rosa_provision.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 - hosts: localhost
   vars:
     cluster_type: rosa
-    ocp_version: "{{ lookup('env', 'OCP_VERSION') | default('4.10.18', True) }}"
+    ocp_version: "{{ lookup('env', 'OCP_VERSION') | default('4.14.5', True) }}"
```
Comment (sanju7216) [inline ibm/mas_devops/playbooks/ocp_rosa_provision.yml:5]: could you please double chekc the version of 4.14.x rosa supports, is it 4.14.12?

Comment (istrate) [reply ibm/mas_devops/playbooks/ocp_rosa_provision.yml:5]: There is no specific versions for ROSA. In their docs they specify guidelines as per https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/policies-and-service-definition#rosa-life-cycle

File: docs/index.md (hunk 2 above)
Comment (istrate) [reply docs/index.md:88]: agree!

File: ibm/mas_devops/playbooks/ocp_rosa_provision.yml (hunk 3 above)
Comment (istrate) [reply ibm/mas_devops/playbooks/ocp_rosa_provision.yml]: Using the latest with as of today is 4.14.16

File: CONTRIBUTING.md (hunk 1 above)
Comment (istrate) [reply CONTRIBUTING.md]: used latest 4.14.16

Comment (sanju7216) [reply CONTRIBUTING.md]: we are using the latest 4.14.16.. Shall we keep it to 4.14.12.. thats what we have tested in fvt and in airgap too

File: ibm/mas_devops/playbooks/ocp_rosa_provision.yml (hunk 3 above)
Comment (sanju7216) [reply ibm/mas_devops/playbooks/ocp_rosa_provision.yml]: we are using 4.14.12 in fvt and airgap mirror.. lets stick with that.

**Review by sanju7216 — CHANGES_REQUESTED** [review]
added comments

**Review by sanju7216 — APPROVED** [review]

**Pull request merged**